CHANGELOG.md

```diff
@@ -1,3 +1,76 @@
+# Thrust 1.11.0
+
+## Summary
+
+Thrust 1.11.0 is a minor release providing bugfixes and performance
+enhancements.
+
+It includes a new sort algorithm that provides up to 2x more performance
+from `thrust::sort` when used with certain key types and hardware.
+
+The new `thrust::shuffle` algorithm has been tweaked to improve the randomness
+of the output.
+
+Our CMake package and build system continue to see improvements with
+improved `add_subdirectory` support, installation rules, status messages, and
+other features that make CUB easier to use from CMake projects.
+
+The release includes several other bugfixes and modernizations, and received
+updates from 12 contributors.
+
+## New Features
+
+- NVIDIA/cub#204: New implementation for `thrust::sort` on CUDA when using
+  32/64-bit numeric keys on SM60+. This improved radix sort algorithm provides
```

Comment:
Can we say "SM60+ (Pascal and up)"? Some of our users will be familiar with our architectures by name only.

Reply:
Updated.